roles/varnish/templates/etc/varnish/varnish.vcl

```diff
@@ -302,301 +295,142 @@ sub vcl_recv {
         elsif (req.url ~ "^/content/((col|m)[0-9]+)") {
             set req.backend_hint = archive_cluster.backend();
         }
-        // all webview
+        # all webview
         elsif (req.url ~ "_escaped_fragment_=" || req.url ~ "^/$" || req.url ~ "^/?.*" || req.url ~ "^/opensearch\.xml" || req.url ~ "^/version\.txt" || req.url ~ "^/search" || req.url ~ "^/contents$" || req.url ~ "^/(contents|data|exports|styles|fonts|bower_components|node_modules|images|scripts)/" || req.url ~ "^/(about|about-us|people|contents|donate|tos|browse)" || req.url ~ "^/(login|logout|workspace|callback|users|publish|robots.txt)") {
```

Comment:
at one point this was a complete "whitelist" of URLs we wanted to serve from webview, allowing a fallback below to legacy. We took that fallback out, and not sure we've been maintaining this list (pretty d*mn sure we have not, actually)

Reply:
Should this just become an `else`?

Reply:
Actually that might be dangerous to change. Current behavior seems to be that anything that does not match this whitelist falls through to legacy_frontend (the first backend in the list).

Reply:
if we've missed anything, I suppose QA will let us know. :-)